dtests/src/test/java/io/snappydata/hydra/cluster/SnappyPrms.java

```diff
@@ -85,6 +85,13 @@
     public static Long useSplitMode;
 
     /**
+     * (boolean) - whether stop mode needs to be checked before deleting the config data if already exists.
+     * This is required in case user wants to start the cluster and then stop the same later on using different script.
+     * In this case, test should not delete the existing configuration data created by previous test.
+     */
+    public static Long isStopMode;
```

Comment:
This is required. As per the scripts(start-servers.sh, start-locators.sh, start-leads.sh) requirement, the configuration data has to be available in respective conf files under product/conf directory. So the config data specific to each test is not actually created in test folder.
In my test run, it creates the required conf files specific to test configurations under the $product/conf directory(e.g. servers, locators and leads files). So, for each test there should be new conf files for different configurations.
If I don't delete the conf files then how would, I create the configurations specific to the current test?

Also, the test config data created by any test run should not be lying under the $product/conf directory even though I don't want to run another test, right?